student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(classroom, age, name, parent_permission)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
```

Comment:
- [ ] Kindly add a method that will serve as a `belongs_to`. A student belongs to a classroom. You can take a look again at this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md) and notice that we need a both-side relationship.

Reply:
done , as advised